web/src/components/datasets/DatasetInfo.tsx

```diff
@@ -58,61 +50,50 @@ type DatasetInfoProps = {
 } & JobFacetsProps &
   DispatchProps
 
-const formatColumnTags = (tags: string[], tag_desc: Tag[]) => {
-  const theme = createTheme(useTheme())
-  return (
-    <>
-      {tags.map((tag, index) => {
-        const tagDescription = tag_desc.find((tagItem) => tagItem.name === tag)
-        const tooltipTitle = tagDescription?.description || 'No Tag Description'
-        return (
-          <MQTooltip title={tooltipTitle} key={tag}>
-            <Chip
-              label={tag}
-              size='small'
-              style={{
-                display: 'row',
-                marginRight: index < tags.length - 1 ? theme.spacing(1) : 0,
-                marginTop: 3,
-              }}
-            />
-          </MQTooltip>
-        )
-      })}
-    </>
-  )
-}
-
 const DatasetInfo: FunctionComponent<DatasetInfoProps> = (props) => {
   const { datasetFields, facets, run, jobFacets, fetchJobFacets, resetFacets } = props
   const i18next = require('i18next')
+  const dsNamespace = useSelector(
+    (state: IState) => state.datasetVersions.result.versions[0].namespace
+  )
+  const dsName = useSelector((state: IState) => state.datasetVersions.result.versions[0].name)
 
-  const [open, setOpen] = useState(false)
-  const [selectedKey, setSelectedKey] = useState<string | undefined>(undefined)
-  const theme = createTheme(useTheme())
+  const loadCollapsedState = () => {
+    const storedState = localStorage.getItem(`dsi_${dsNamespace}_${dsName}`)
+    return storedState ? JSON.parse(storedState) : []
+  }
 
   useEffect(() => {
     run && fetchJobFacets(run.id)
-    run && fetchTags()
   }, [run])
 
-  // unmounting
   useEffect(
     () => () => {
       resetFacets()
     },
     []
   )
+  const [expandedRows, setExpandedRows] = useState<number[]>(loadCollapsedState)
+
+  const toggleRow = (index: number) => {
+    setExpandedRows((prevExpandedRows) => {
+      const newExpandedRows = prevExpandedRows.includes(index)
+        ? prevExpandedRows.filter((rowIndex) => rowIndex !== index)
+        : [...prevExpandedRows, index]
 
-  const tagData = useSelector((state: IState) => state.tags.tags)
-  const handleOpen = (key: string) => {
-    setOpen(true)
-    setSelectedKey(key)
+      localStorage.setItem(`dsi_${dsNamespace}_${dsName}`, JSON.stringify(newExpandedRows))
```

Comment:
I kind of want to get away from doing this because it doesn't move across urls. Could we possibly use use parameters for functionality like this?